tests/e2e.sh

```diff
@@ -31,8 +31,6 @@ SUDO=${SUDO:-}
 OS=$(uname -s | awk '{print tolower($0)}')
 OS=${OS:-linux}
 
-EXCLUDED_RESOURCE_REGEX="verticalpodautoscaler"
```

Comment:
Hmm wonder why we excluded this? cc @brancz

Reply:
I believe it's because of verticalpodautoscaler compatibility issues with minikube. If we move to KIND, we might not have this anymore.

Reply:
It's not easy to create `verticalpodautoscaler` in test environment and we can't get the metrics of it. So it should be excluded :)